doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -612,52 +643,107 @@ public void toPublicationSetsUrlInPublicationContextWhenCrossrefDocumentHasLink(
         assertThat(actualLink.toString(), is(equalTo(SAMPLE_LINK)));
     }
 
+    @Test
+    @DisplayName("toPublication handles all interesting and required fields in CrossrefDocument for journal-article")
+    public void toPublicationHandlesAllInterestingAndRequiredFieldsInCrossrefDocumentForJournalArticle()
+            throws InvalidIssnException, InvalidIsbnException {
+
+        CrossRefDocument crossRefDocument = createSampleDocumentJournalArticle();
+        Publication actualPublication = toPublication(crossRefDocument);
+        assertRequieredValuesAreConverted(actualPublication);
+    }
 
     @Test
     @DisplayName("toPublication handles all interesting and required fields in CrossrefDocument for book")
     public void toPublicationHandlesAllInterestingAndRequiredFieldsInCrossrefDocumentForBook()
             throws InvalidIssnException, InvalidIsbnException {
-        CrossRefDocument crossRefDocument = createCompleteCrossRefDocument();
 
+        CrossRefDocument crossRefDocument = createSampleDocumentBook();
         Publication actualPublication = toPublication(crossRefDocument);
+        assertRequieredValuesAreConverted(actualPublication);
+    }
 
-        assertAssignedValuesAreConverted(actualPublication, sampleCrossrefDateToInstant());
+    @Test
+    @DisplayName("toPublication handles all interesting and required fields in CrossrefDocument for book-chapter")
+    public void toPublicationHandlesAllInterestingAndRequiredFieldsInCrossrefDocumentForBookChapter()
+            throws InvalidIssnException, InvalidIsbnException {
+
+        CrossRefDocument crossRefDocument = createSampleDocumentBookChapter();
+        Publication actualPublication = toPublication(crossRefDocument);
+        assertRequieredValuesAreConverted(actualPublication);
     }
 
-    private CrossRefDocument createCompleteCrossRefDocument() {
-        CrossRefDocument crossRefDocument = createSampleDocumentBook();
-        CrossrefDate crossrefDate = createCrossrefDate();
 
+    @Test
```

Comment:
These tests seem to enjoy their whitespace

Reply:
Killed a lot of white-space...